common/persistence/tests/cassandra_test.go

```diff
@@ -173,3 +172,9 @@ func TestCassandraClusterMetadataPersistence(t *testing.T) {
 	s.TestBase.Setup(nil)
 	suite.Run(t, s)
 }
+
+func TestDynamicRateLimitedCassandra(t *testing.T) {
```

Comment:
Related to the other comment above. don't think this is specific to cassandra and needs a real persistence connection.